### PR TITLE
Initial code cleanup & Experimental page

### PR DESCRIPTION
**Pull request by AVHopp**
This PR performs a small initial code clean up and is also meant to see how well we get along with doing PRs.

It does the following major things:
- Reformatting of the pages code according to my local environments and rules: This is automatically configured by my IDE, I hope this does not cause any trouble
- Removing several unused imports
- Moving and actually using the code that was already contained in the utility
- Make git ignore the folder for the model weights

Note that I have tested this branch locally, and the streamlit app worked for me, but I cannot guarantee that the logic (e.g. the new import of the utils) also works for the deployed app since I do not know how (or if) I can even test that.

### CODE REVIEW

**Comment by AVHopp** [comment]
@dev-csftan unfortunately, I cannot assign you as a reviewer (I guess the repository would need some more configuration for that?) hence a quick ping here :)

**Comment by dev-csftan** [comment]
> @dev-csftan unfortunately, I cannot assign you as a reviewer (I guess the repository would need some more configuration for that?) hence a quick ping here :)
Alex, I can add you as a contributor as well.

**Review by dev-csftan — COMMENTED** [review]
@dezhi0730 please double-check this, I remeber that you said some duplicates are due to the working mode in the app service

**Comment by dezhi0730** [comment]
> @AVHopp hi, Alex, some of these function duplications are on purpose by Eason cause he encounterd "the function not found problem" in the uptimize appservice env. @dezhi0730 you can describe the problem you encounterd by giving a screenshop.

hi, Alex, the main issue is that because of a path problem, it can't find utils.py, although it runs fine in my local environment. I'm also a bit unclear about the details. I plan to test the newly revised code on Uptimize Appservice env  later and then provide more detailed information.

**Comment by AVHopp** [comment]
@dezhi0730 @dev-csftan Ok great, just keep me updated :) We can easily also revert these changes again if it turns out that another format resp. paths are necessary.

**Comment by AVHopp** [comment]
@dev-csftan @dezhi0730 @S0upreme I have added the experimental page. I will demonstrate it in our next meeting, but feel free to review already.

**Review by dezhi0730 — APPROVED** [review]

File: src/utils.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import os
```
Comment (AVHopp) [inline src/utils.py:1]: Since @dezhi0730 approved the PR, does this mean that the code works in our app?

Comment (dev-csftan) [reply src/utils.py:1]: yes, he tested, @AVHopp

**Pull request merged**